temperature/crud.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+from fastapi import HTTPException, Depends, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.exc import IntegrityError
+from websockets.legacy import async_timeout
+
+from temperature import utils
+from weather import models
+from weather.crud import get_all_cities
+from dependencies import get_db
+
+
+async def get_temperature_list(db: AsyncSession):
+    result = await db.execute(
+        select(models.Temperature)
+    )
+    return result.scalars()
```

Comment:
The `get_temperature_list` function returns a result from the database. Ensure that the calling function or endpoint properly handles the asynchronous generator returned by `scalars()`.